Runtime/JSArrayBuffer.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+
+namespace NodeApi;
+
+public struct JSArrayBuffer
+{
+    private JSValue _value;
+
+    public static implicit operator JSValue(JSArrayBuffer value) => value._value;
+    public static implicit operator JSObject(JSArrayBuffer value) => (JSObject)value._value;
+
+    public static explicit operator JSArrayBuffer(JSValue value) => new() { _value = value };
+    public static explicit operator JSArrayBuffer(JSObject value) => (JSArrayBuffer)(JSValue)value;
+
+    public JSArrayBuffer(int byteLength) =>
+        _value = JSValue.CreateArrayBuffer(byteLength);
+
+    public JSArrayBuffer(ReadOnlySpan<byte> data)
+        => _value = JSValue.CreateArrayBuffer(data);
+
+    public JSArrayBuffer(object? external, ReadOnlyMemory<byte> data)
+        => _value = JSValue.CreateExternalArrayBuffer(external, data);
+
+    public JSArrayBuffer(ReadOnlyMemory<byte> data)
+        => _value = JSValue.CreateExternalArrayBuffer(null, data);
+
+    public Span<byte> Data => _value.GetArrayBufferInfo();
+
+    public int ByteLength => _value.GetArrayBufferInfo().Length;
```

Comment:
ArrayBuffers always use bytes as the element type, so I think just `Length` would be good.

I assume we'll add separate typed array structs later.

Reply:
Good point! I just started to work on this class and it is just a draft. I will change it. I must finish the `JSException` first.

Reply:
Changed to `Length`.